BLSQ/iaso
# IA-3178 : Have pictures for ALL form submissions (for better demo/Visualization)

Related JIRA tickets : [IA-3178](https://bluesquare.atlassian.net/browse/IA-3178)

## Self proofreading checklist



## Changes

Added pictures on submissions for other org unit types than health facilities!

## How to test

From setuper folder, launch `python3 setuper.py` to create data and open submissions linked to the form **Equipment/Pop/Social mob./Microplans** for other type than health facility : http://localhost:8081/dashboard/forms/submissions/accountId/1/formIds/3/tab/files

You should see differents pictures for each org unit type.

## Print screen / video

[Pictures_by_org_unit_type.webm](https://github.com/user-attachments/assets/d414d6a7-b94a-47c6-8610-3f21b72fbb67)


[IA-3178]: https://bluesquare.atlassian.net/browse/IA-3178?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
